web/src/test/manual/embed/android-harness/build.sh

```diff
@@ -10,19 +10,50 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 
 THIS_DIR="$(dirname "$THIS_SCRIPT")"
```

Comment:
`$THIS_SCRIPT_PATH` is already defined.

```suggestion
cd "$THIS_SCRIPT_PATH"
```